modules/acquia_cms_person/config/optional/views.view.people.yml

```diff
@@ -282,81 +306,55 @@ display:
             default_group: All
             default_group_multiple: {  }
             group_items: {  }
-          plugin_id: search_api_language
-      sorts:
-        title:
-          id: title
-          table: search_api_index_content
-          field: title
-          relationship: none
-          group_type: group
-          admin_label: ''
-          order: ASC
-          exposed: false
-          expose:
-            label: ''
-          plugin_id: search_api
-      title: People
+      style:
+        type: default
+      row:
+        type: search_api
+        options:
+          view_modes:
+            'entity:node':
+              article: default
+              event: default
+              page: default
+              person: card
+              place: default
+      query:
+        type: search_api_query
+        options:
+          bypass_access: false
+          skip_access: false
+          preserve_facet_query_args: false
+      relationships: {  }
       header: {  }
       footer: {  }
-      empty:
-        view_fallback:
-          id: view_fallback
-          table: views
-          field: view_fallback
-          relationship: none
-          group_type: group
-          admin_label: ''
-          empty: true
-          view_to_insert: 'people_fallback:default'
-          inherit_arguments: 0
-          plugin_id: view_fallback
-        main_listing_pages_view:
-          id: main_listing_pages_view
-          table: views
-          field: main_listing_pages_view
-          relationship: none
-          group_type: group
-          admin_label: ''
-          empty: true
-          tokenize: 0
-          content:
-            value: 'Sorry, no content found. Please try to search with different keywords.'
-            format: filtered_html
-          plugin_id: main_listing_pages_view
-      relationships: {  }
-      arguments: {  }
       display_extenders: {  }
     cache_metadata:
       max-age: -1
       contexts:
-        - 'languages:language_content'
         - 'languages:language_interface'
         - url.path
         - url.query_args
         - 'user.node_grants:view'
         - user.permissions
       tags:
-        - 'config:field.storage.node.field_person_type'
```

Comment:
@apathak18 - Why we are removing this config dependency

Reply:
Added back.